src/views.rs

```diff
@@ -22,7 +19,7 @@ use std::{
 /// ```
 pub struct Views {
     source_type_id: TypeId,
-    view_casters: Arc<boxcar::Vec<DynViewCaster>>,
+    view_casters: Box<boxcar::Vec<DynViewCaster>>,
```

Comment:
I seem to be missing something here, replacing the `Arc`, either with nothing or with a `Box` regresses performance even though `view_casters` is never cloned?